spyder/plugins/ipythonconsole/widgets/shell.py

```diff
@@ -186,8 +186,14 @@ def set_color_scheme(self, color_scheme, reset=True):
             self.reset(clear=True)
         if not dark_color:
             self.silent_execute("%colors linux")
```

Comment:
What does this line do?

Reply:
We use this to change the colors of the tracebacks

![image](https://user-images.githubusercontent.com/16781833/64038260-3d08fc80-cb1d-11e9-9965-bf017aad9668.png)

[Here the docs for further info](https://ipython.readthedocs.io/en/stable/config/details.html?#terminal-colors)

Reply:
@dalthviz could we add this comment to the code :-) ?